test/powershell/Modules/Microsoft.WSMan.Management/CredSSP.Tests.ps1

```diff
@@ -0,0 +1,105 @@
+Describe "CredSSP cmdlet tests" -Tags 'Feature','RequireAdminOnWindows' {
+
+    BeforeAll {
+        $powershell = Join-Path $PSHOME "powershell"
+
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if ( ! $IsWindows ) {
+            $PSDefaultParameterValues["it:skip"] = $true
+        }
+
+        $notEnglish = $false

```

Comment:
As a general rule when using this pattern, a test that is skipped in an environment should not execute any code.

Lines 11 through 15 should not be executed if the test is skipped. Putting them in an else {} would solve this problem.